package.json

```diff
@@ -10,6 +10,9 @@
         "@babel/preset-env": "^7.16.8",
         "@typescript-eslint/eslint-plugin": "^6.18.1",
         "@typescript-eslint/parser": "^6.18.1",
+        "@vitejs/plugin-vue": "^5.0.4",
```

Comment:
We should call out that this requires a higher node version. `createproject` failed for me this morning with:

```
[2/4] 🚚  Fetching packages...
error @vitejs/plugin-vue@5.0.4: The engine "node" is incompatible with this module. Expected version "^18.0.0 || >=20.0.0". Got "16.20.2"
error Found incompatible module.
```

Reply:
oh, good catch